internal/app/admin/http/cache_test.go

```diff
@@ -699,6 +699,332 @@ func TestAdminServer_CacheDumpHandler_WildcardSuffix_NotFound(t *testing.T) {
 	}
 }
 
+func testAdminServerClearCacheHelper(t *testing.T, urls []string) {
```

Comment:
Seems that there are some code duplication between these 3 functions. The main difference is the TypeUrl. Could some of the logic be consolidated into a "base" test function:

Reply:
I think the challenge is that it isn't just the TypeUrl. Since these tests check between v2 and v3, most variables, while having the same function, are still different types. I do think I can consolidate one of these tests a bit more, and will do so in next commit